pages/api/docs/getAllRepos.ts

```diff
@@ -28,7 +31,8 @@ const getUserRepositoriesHandler = async (req: NextApiRequest, res: NextApiRespo
 	}
 	const eventProperties = { response_status: 200, result_length: repoList.length };
 	rudderStackEvents.track(session?.user?.id ?? "absent", "", 'user-repos-from-providers', { type: 'HTTP-200', eventStatusFlag: 1, eventProperties });    
-	return res.status(200).json({ repoList: repoList });
+  	res.status(200).json({ repoList: repoList });
+	await saveRepoIdentifierToDb(repoList);
```

Comment:
Ideally this should only run for github right? since in case of bitbucket we are already saving the whole repo obj in db inside the `getUserRepositories` function. Although, we can still ignore it since in this insert query we `Do Nothing` in case of a conflict. 
LOW LOGAF: DB query will run unnecessarily for bitbucket.